cmd/network/verification.go

```diff
@@ -176,13 +178,22 @@ func (e *EgressVerification) Run(ctx context.Context) {
 		out := onv.ValidateEgress(c, *inputs[i])
 		out.Summary(e.Debug)
 
-		// Only suggest sending a service log if the failures are egress-url related
+		// Move cluster into LS if in case cluster can't connect with PD(OR)DMS || service log if the failures other egress-url related
 		if !out.IsSuccessful() && len(out.GetEgressURLFailures()) > 0 {
 			postCmd := generateServiceLog(out, e.ClusterId)
-			if err := postCmd.Run(); err != nil {
+
+			bUrl := strings.Join(postCmd.TemplateParams, ",")
```

Comment:
**nitpick:** we could give this parameter a proper name and re-use it in the SL sent / LS sent.